### PR TITLE
Add MangaDig and make ColaManga a multisrc

### LINKED ISSUE #304: Mangadig
### Source name

Mangadig

### Source link

https://mangadig.com/

### Source language

English

### Other details

It does have 18+/NSFW content along with regular content 

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/keiyoushi/extensions-source/) and verified it does not appear in the code base.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by beer-psi**
Closes #304

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    final override val name: String,
+    final override val baseUrl: String,
+    final override val lang: String,
+) : ParsedHttpSource(), ConfigurableSource {
+
+    override val supportsLatest = true
+
+    private val json: Json by injectLazy()
+
+    private val intl = ColaMangaIntl(lang)
```
Comment (AwkwardPeak7) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt:47]: why not `:lib:i18n`

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+
+    override fun popularMangaFromElement(element: Element) = SManga.create().apply {
+        element.selectFirst("a.fed-list-title")!!.let {
+            setUrlWithoutDomain(it.attr("href"))
+            title = it.text()
+        }
+        thumbnail_url = element.selectFirst("a.fed-list-pics")?.absUrl("data-original")
+    }
+
+    override fun popularMangaNextPageSelector() = "a:contains(下页):not(.fed-btns-disad)"
```
Comment (AwkwardPeak7) [inline multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt]: this selector works on the english version?

Comment (beer-psi) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt]: fixed

File: multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt (hunk 1 above)
Comment (beer-psi) [reply multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/colamanga/ColaManga.kt:47]: the multisrc generator doesn't copy `assets`, only `src` and `res`.

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**